influxdb/__init__.py

```diff
@@ -0,0 +1,148 @@
+
+
+#!/usr/bin/env python3
+# vim: set encoding=utf-8 tabstop=4 softtabstop=4 shiftwidth=4 expandtab
+#########################################################################
+#  Copyright 2016 Kai Meder <kai@meder.info>
+#########################################################################
+#  This file is part of SmartHomeNG.
+#
+#  SmartHomeNG is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  SmartHomeNG is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with SmartHomeNG. If not, see <http://www.gnu.org/licenses/>.
+#########################################################################
+
+import logging
+import socket
+import json
+from lib.model.smartplugin import SmartPlugin
+
+
+class InfluxDB(SmartPlugin):
+    PLUGIN_VERSION = "1.0.0"
+    ALLOW_MULTIINSTANCE = False
+
+    def __init__(self, smarthome, host='localhost', udp_port=8089, keyword='influxdb', tags={}, fields={}, value_field='value'):
+        self.logger = logging.getLogger(__name__)
+        self.logger.info('Init InfluxDB')
+
+        self.host = host
+        self.udp_port = udp_port
+        self.keyword = keyword
+        self.tags = tags
+        self.fields = fields
+        self.value_field = value_field
+        self.item_config = {}
+
+    def run(self):
+        self.alive = True
+
+    def stop(self):
+        self.alive = False
+
+    def parse_item(self, item):
+        if self.keyword in item.conf or 'influxdb_name' in item.conf or 'influxdb_tags' in item.conf or 'influxdb_fields' in item.conf:
```

Comment:
Checking and getting item config values can be done using `get_iattr_value` and `has_iattr_value` of `SmartPlugin`.